pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -17,7 +17,7 @@
  *
  */
 
-//go:generate deepcopy-gen -i . --go-header-file ../../../../hack/boilerplate/boilerplate.go.txt
+//go:generate deepcopy-gen -v 1 -i . --go-header-file ../../../../hack/boilerplate/boilerplate.go.txt
```

Comment:
leftovers ? or needed?

Reply:
We don't need this line since we are no longer using go generate for deepcopy-gen thanks.